packages/openai-compatible/src/openai-compatible-chat-language-model.ts

```diff
@@ -385,9 +385,9 @@ export class OpenAICompatibleChatLanguageModel implements LanguageModelV1 {
       stream: true,
 
       // only include stream_options when in strict compatibility mode:
-      stream_options: this.config.includeUsage
-        ? { include_usage: true }
-        : undefined,
```

Comment:
we use json stringify, which omits undefined values from objects, so how is this any different in terms of the body that is sent?

Reply:
i understand that. but the actual body that is sent will be the same because we json stringify later